modules/static/prometheus/client-model/initref

```diff
@@ -0,0 +1 @@
+147c58e9608a4f9628b53b6cc863325ca746f63a
```

Comment:
This is the first tag in the public BSR: https://buf.build/prometheus/client-model/commits/main